spec.yaml

```diff
@@ -759,3 +762,59 @@ paths:
           description: Error
           schema:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
+
+  /v4/releases/:
+    get:
+      operationId: getReleases
+      tags:
+        - releases
+      summary: Get releases
+      description: |
+        Lists all releases available for new clusters or for upgrading existing
+        clusters. Might also serve as an archive to obtain details on older
+        releases.
+      parameters:
+        - $ref: 'parameters.yaml#/parameters/OrganizationIdPathParameter'
+      responses:
+        "200":
+          description: Releases list
+          schema:
+            type: array
+            items:
+              $ref: "definitions.yaml#/definitions/V4ReleaseListItem"
+          examples:
+            application/json:
+              [
+                {
+                  "version": "1.14.9",
+                  "timestamp": "2017-09-21T08:14:03.37759Z",
+                  "description": "Minor updates of kubernetes and calico, both for security fixes.",
```

Comment:
We have a list of changelogs available. I think it would be good to have then a list of descriptions. Our internal code requires to have one changelog (description) for each component that changed. See

- https://github.com/giantswarm/versionbundle/blob/efdbdae719c56ab4880f8fe56643d60a76fc19e5/changelog.go- https://github.com/giantswarm/versionbundle/blob/efdbdae719c56ab4880f8fe56643d60a76fc19e5/changelog_test.go

Reply:
I like the structured approach, which would enable us to automatically generate these. I'm open to think about additional field(s) to allow for more prosaic descriptions of a release to add to user friendliness

Reply:
So we need a `description` field per component. What if that component hasn't changed since the last release? Make the description `null` comes to my mind.

Reply:
I think it's semantically much more useful when there is a description per component. The description in the release item serves as the optional additional way to add human-friendly information.